src/test/java/io/beanmapper/testmodel/enums/Day.java

```diff
@@ -0,0 +1,24 @@
+package io.beanmapper.testmodel.enums;
+
+public enum Day {
+
+    MONDAY("Monday"),
+    TUESDAY("Tuesday"),
+    WEDNESDAY("Wednesday"),
+    THURSDAY("Thursday"),
+    FRIDAY("Friday"),
+    SATURDAY("Saturday"),
+    SUNDAY("Sunday");
+
+    private final String displayName;
+
+    Day(String displayName) {
+        this.displayName = displayName;
+    }
+
+    @Override
+    public String toString() {
+        return this.name().toLowerCase();
```

Comment:
Is the `displayName` not intended to be returned here? If not, what is the purpose of displayName ?

Reply:
You're absolutely right. Head must've checked out already, when I wrote that. I'll go ahead and change that, and the tests, to use the `displayName` variable. Thanks you very much!